src/Sylius/Bundle/SettingsBundle/Model/Settings.php

```diff
@@ -1,119 +1,105 @@
 <?php
 
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
 namespace Sylius\Bundle\SettingsBundle\Model;
 
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
 /**
- * @author Paweł Jędrzejewski <pawel@sylius.org>
+ * @author Steffen Brem <steffenbrem@gmail.com>
  */
-class Settings implements \ArrayAccess
+class Settings implements SettingsInterface
 {
     /**
-     * @var array
+     * @var mixed
      */
-    protected $parameters;
+    protected $id;
 
     /**
-     * @param array $parameters
+     * @var string
      */
-    public function __construct(array $parameters)
-    {
-        $this->parameters = $parameters;
-    }
+    protected $schema;
```

Comment:
We can have in Sylius Schemas object. For this reason this name can be confusing. Maybe the better will be ``schemaName``, WDYT? :)